client/deployments/client.go

```diff
@@ -157,6 +191,87 @@ func listArtifact(a artifactData, detailLevel int) {
 	fmt.Println("--------------------------------------------------------------------------------")
 }
 
+func (c *Client) DirectUpload(
```

Comment:
this is a copy-paste (almost) from the func below on purpose, they are similar now, but it is possible they will diverge, so I wanted to keep them separate.